libqtile/utils.py

```diff
@@ -194,9 +194,8 @@ def send_notification(title, message, urgent=False, timeout=10000, id=None):
     https://developer.gnome.org/notification-spec/
     """
     if not has_dbus:
-        logger.warning(
-            "dbus-next is not installed. Unable to send notifications."
-        )
+        from libqtile.popup import send_popup
+        send_popup(f"<b>{title}</b>\n{message}", timeout=timeout)
```

Comment:
Can we make this configurable? I basically never want popups from any application ever :)

Reply:
Do you ever get notifications through `send_notification`? You'll likely only ever see this if your config is wrong when you try to load it. I don't think you'll be using `send_notification` at all currently unless you have a notification server running, in which case you'll not get any popups.

Reply:
No, I don't run any libnotify servers or anything like that. I guess what I'm worried about is people starting to use send_notification() for other stuff and that I'll start getting popups all of the sudden. But I can complain then, instead of now :)

Reply:
Please do -- I do not want `send_notification` to become more than something explicitly triggered by custom user code. E.g. in config functions or the one or two `Qtile.cmd_` methods that use it.

Reply:
I'm changing this so rather than send_notification falling back to send_popup, Qtile.load_config will itself deal with the fallback, so there is no chance of popups appear on account of having no libnotify server running. Otherwise like you hint at, popups will be seen as a fallback for these but it's only for this one case :)